src/graphics/vulkan/vulkan_graphics.cpp

```
@@ -479,7 +479,42 @@ namespace moth_graphics::graphics::vulkan {
+        float fitHeight = static_cast<float>(physical.height);
+        if (physicalAspect > logicalAspect) {
+            fitWidth = fitHeight * logicalAspect;
+        } else {
+            fitHeight = fitWidth / logicalAspect;
+        }
+        float const offsetX = (static_cast<float>(physical.width) - fitWidth) * 0.5f;
+        float const offsetY = (static_cast<float>(physical.height) - fitHeight) * 0.5f;
+
         auto& commandBuffer = context->m_target->GetCommandBuffer();
+
+        VkViewport viewport;
```

Clang (14.0.6)

[error] variable 'viewport' is not initialized

(cppcoreguidelines-init-variables,-warnings-as-errors)